metadata.yaml

```diff
@@ -13,3 +13,6 @@ releaseSeries:
 - major: 0
   minor: 6
   contract: v1beta1
+- major: 0
```

Comment:
This change is technically not related to a new API Version, but to a new release series. Should we add this now or right before cutting a release branch?

Reply:
I'm fine with both and have also not a hard opinion on this.
If we add it now, the `clusterctl` request for a CAPO 0.7 release will fail anyway as we haven't released it yet.

Reply:
But as we maybe want to graduate this version to v1beta1 it‘s better to remove it